src/main/scala/org/constellation/consensus/EdgeProcessor.scala

```diff
@@ -45,7 +46,10 @@ case class FinishedCheckpointResponse(isSuccess: Boolean = false)
 case class TipData(checkpointBlock: CheckpointBlock, numUses: Int, height: Height)
 
 case class Snapshot(lastSnapshot: String, checkpointBlocks: Seq[String], publicReputation: SortedMap[Id, Double])
-    extends Signable
+    extends Signable {
+  override def getEncoding: String = hashSerialized(lastSnapshot :+ checkpointBlocks)
+  override def getHexEncoding: String = KeyUtils.bytes2hex(hashSerializedBytes(lastSnapshot :+ checkpointBlocks))
```

Comment:
You can create `val toEncode = lastSnapshot :+ checkpointBlocks` and then reuse that